### ISSUE TITLE
It can't get amount when upgrading from v1.2.0 to v2.0.0 on okc test

### ISSUE THREAD

**Issue by zjg555543** [spoo-bar/cosmy-wasmy#49]
**Describe the bug**
I have testing on okc testnet.After i upgrading v2.0.0, i can't see my amount of acount.
**v1.2.0:**
![image](https://user-images.githubusercontent.com/7919751/228726233-164c1df7-41cf-40bd-af2e-8412d5a1736d.png)

**v2.0.0**
![image](https://user-images.githubusercontent.com/7919751/228726433-55ebcead-abf0-47ce-a7c4-128328264357.png)

**To Reproduce**
1 run okc localnet on macos
```
git clone https://github.com/okx/exchain.git
cd exchain/dev/
./start.sh
```
2 config the chain config of cosmy-wasmy 
```
{
            "configName": "local okc with okt",
            "chainId": "exchain-67",
            "chainEnvironment": "localnet",
            "addressPrefix": "ex",
            "rpcEndpoint": "http://localhost:26657",
            "defaultGasPrice": "0.0000000001",
            "chainDenom": "okt"
},
```
3 create a random account X in comsy-wasmy , e.g: ex1kyh26rw89f8a4ym4p49g5z59mcj0xs4j9xcvw3

4 transfer okt to the account X on localnet
```
exchaincli tx send ex1h0j8x0v9hs4eq6ppgamemfyu4vuvp2sl0q9p3v X(replace your account) 10okt --from captain --gas auto  --gas-prices 0.0000000001okt --gas-adjustment 1.3 -y -b block

```
5 switch the version between 1.2.0 and 2.0.0, and restart vscode

**Expected behavior**
See the acount amount on version v2.0.0

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS]
 - Version [e.g. 12.6.2]
 - Running vscode Insider [e.g no]

**Additional context**

**Comment by zjg555543** [comment]
Thanks~
1. Yes, i'm sure the rpc node was running. I switched and rebooted versions repeatedly, v1.2.0 is ok
2. I had refresh the account many times with v2.0.0, but it's not  unable to display

**Comment by zjg555543** [comment]
I got the reason. It is succeed when I rollback the version of cosmjs from 0.29.5 to 0.28.4 .

**Comment by spoo-bar** [comment]
I think it was that new cosmjs version dependencies weren't handled properly. 
I have published new version `v2.1.0` of the extension with the fix. Can you check it out if this fixes the issue?

**Comment by zjg555543** [comment]
Hi~ I have commit PR https://github.com/spoo-bar/cosmy-wasmy/pull/55 to support OKTChain, please review it, Thanks~
